server/src/routes/api/v1/voteRouter.js

```diff
@@ -0,0 +1,44 @@
+import express from "express";
+import { Review, Vote } from "../../../models/index.js";
+import { ValidationError } from "objection";
+import ReviewSerializer from "../../../serializers/ReviewSerializer.js"
+
+const voteRouter = new express.Router();
+
+voteRouter.post("/", async (req, res) => {
+    const voteValue = req.body.voteValue;
+    const reviewId = req.body.reviewId;
+    const userId = req.user.id;
+    try {
+        const voteExists = await Vote.query().where("reviewId",`${reviewId}`).where("userId",`${userId}`)
+        if (voteExists.length>0) {
+            if(voteValue === voteExists[0].voteValue) {
+                await Vote.query().delete().where("reviewId",`${reviewId}`).where("userId",`${userId}`)
```

Comment:
since you will have already retrieved the vote on line 13, use `deleteById` https://vincit.github.io/objection.js/api/query-builder/mutate-methods.html?#deletebyid
remove string interpolation